modules/installation/proc-rhdh-deploy-aks-operator.adoc

```diff
@@ -62,7 +62,7 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: rhdh-ingress
-  namespace: <your_namespace>
+  namespace: {my-product-namespace}
```

Comment:
Line 59: `[source,yaml,subs="+attributes,+quotes"]`

Reply:
See: https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/#incremental